app/model/model.scala

```diff
@@ -62,7 +62,7 @@ package object model {
       case BcPngr => "passengers"
       case CapitalGainsTax => "capital-gains-tax"
       case Mib => "merchandise-in-baggage"
-      case Ni => "northern-ireland"
+      case NiEuVatOss | PfNiEuVatOss => "northern-ireland"
```

Comment:
These are probably better specified as `vat-oss`.

Reply:
That's what they are in the library ?

Reply:
Nm I see the name gets changed for the origin